data/migrations/deb/disabled_1_0_97_to_1_0_96.py

```diff
@@ -12,8 +12,8 @@
 from storage import store_utils
 from storage.chunked_file_store import ChunkedFileStore
 
-from sovrin_common.config_util import getConfig
-from sovrin_common.txn_util import getTxnOrderedFields
+from indy_common.config_util import getConfig
```

Comment:
There should be no changes in migration scripts since they are supposed to be used with an old version of code.

Reply:
Migration scripts should be operable on the target code base.